frontend/src/metabase/ui/components/typography/List/List.styled.tsx

```diff
@@ -0,0 +1,18 @@
+import type { MantineThemeOverride } from "@mantine/core";
+
+export const getListOverrides = (): MantineThemeOverride["components"] => ({
+  List: {
+    styles: theme => {
+      return {
+        root: {
+          // to revert "none" from the reset
+          listStyleType: "revert",
+        },
+        item: {
+          lineHeight: "1.5",
+          color: theme.fn.themeColor("text-dark"),
```

Comment:
Can I still override text color on a per usage basis with this here?

Reply:
Out of the box `List.Item` doesn't support the color, but we can customize a single item with either custom css/styled/inline style